registry.json

```diff
@@ -1,4 +1,18 @@
 [
+    {
+    "ticker": "TETON",
+    "poolId": "a2a7050e1ef0618dc76bdcc9bbba0c01f88fe1b88e19cc3218acacd8",
+    "social": {
+      "website": "https://tetonpool.com/",
+      "twitter": "RockingHatLLC",
+      "facebook": "",
+      "instagram": "",
+      "youtube": "" 
+      },
+    "telegram": "RockingHat",
+    "operatorName": "Rocking Hat LLC",
```

Comment:
Please can you change the operator name to your real full name, real first name or leave blank?